management/server/peer.go

```diff
@@ -56,7 +56,7 @@ func (p *Peer) Copy() *Peer {
 }
 
 //GetPeer returns a peer from a Store
-func (am *AccountManager) GetPeer(peerKey string) (*Peer, error) {
+func (am *DefaultAccountManager) GetPeer(peerKey string) (*Peer, error) {
```

Comment:
I suggest keeping the AccountManager interface here, then we can easily replace it with some other implementation if needed.
It will also be easier to mock in tests
That is the whole purpose of interfaces :)